### ISSUE TITLE
Feature request: hover over passive tree node to see name

### ISSUE THREAD

**Issue by xrisk** [HeartofPhos/exile-leveling#68]
As a new player, I often get confused by the tree and end up having to cross-check with pob to ensure I’m not making a mistake. I understand that this is not too important, but it would be nice to have depending on how much effort is required to implement it.

```
<circle cx="-7403" cy="-3011" id="n27235" r="50" class="mastery">
<title>Recovery Mastery</title>
</circle>
```

I see this information is already present in the SVG. Perhaps a simple text element that updates when you mouseover?

**Comment by xrisk** [comment]
NVM, I’m stupid: this is already implemented I didn’t realize that it takes like a second to show up. 

Thank you for the great work!

**Comment by HeartofPhos** [comment]
While this does work as you've stated it's pretty poor UX and I plan on making it more responsive, will likely be updated in a future UI/UX rework

**Comment by HeartofPhos** [comment]
resolved by #94
